test/integration/README.rst

```diff
@@ -0,0 +1,13 @@
+************************************
+aws-encryption-sdk Integration Tests
+************************************
+
+In order to run these integration tests successfully, these things which must be configured.
```

Comment:
Minor: wording polish ("these things which must be configured")

Reply:
Updated to "these things must be configured".